src/hotspot/share/code/nmethod.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 1997, 2025, Oracle and/or its affiliates. All rights reserved.
+ * Copyright 2025 Arm Limited and/or its affiliates.
```

Comment:
Still questionable, here. Do we have clear guidelines?

Reply:
Thanks for the feedback. I followed the current guideline at https://openjdk.org/guide/#copyright-headers which advises:

> If your affiliation doesn’t have a copyright notice, again consult your legal representative to see if you should add one.

If the guideline isn't applicable or if there is a more detailed guidance on when Copyright header should or should not be added, I'd appreciate further advice.

Reply:
Many companies have contributed to this file over time. If we all added our copyright lines, the header would become unreadable. The usual OpenJDK practice is to only update header for new files or very substantial rewrites.